### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by szepeviktor**
Would you like to code as per PSR-12?

### CODE REVIEW

**Comment by georgeboot** [comment]
I'd say yes

**Comment by szepeviktor** [comment]
> I'd say yes

Done 🍏

**Comment by szepeviktor** [comment]
Adding a few unit tests seems like a good idea.

File: src/StorageDirectories.php [hunk 1]
```diff
@@ -9,7 +9,7 @@ class StorageDirectories
      *
      * @var string
      */
-    public const Path = '/tmp/storage';
+    public const PATH = '/tmp/storage';
```
Comment (tillkruss) [inline src/StorageDirectories.php:12]: Is this PSR-12?

File: stubs/runtime.php [hunk 2]
```diff
@@ -37,7 +35,7 @@
     MaintenanceMode::setUp();
 
     $kernel = $app->make(ConsoleKernel::class);
-    $status = $kernel->handle($input = new ArgvInput, new ConsoleOutput);
+    $status = $kernel->handle($input = new ArgvInput(), new ConsoleOutput());
```
Comment (tillkruss) [inline stubs/runtime.php:38]: Is this PSR-12?

File: stubs/runtime.php [hunk 3]
```diff
@@ -7,10 +7,8 @@
 use CacheWerk\BrefLaravelBridge\Queue\QueueHandler;
 use CacheWerk\BrefLaravelBridge\StorageDirectories;
 use CacheWerk\BrefLaravelBridge\Octane\OctaneClient;
-
```
Comment (tillkruss) [inline stubs/runtime.php:10]: Can we disable this rule?

File: src/StorageDirectories.php (hunk 1 above)
Comment (szepeviktor) [reply src/StorageDirectories.php:12]: phpcs told me it is

File: stubs/runtime.php (hunk 2 above)
Comment (szepeviktor) [reply stubs/runtime.php:38]: Run `composer exec -- phpcs --standard=PSR12 -s src/ stubs/`

File: stubs/runtime.php (hunk 3 above)
Comment (szepeviktor) [reply stubs/runtime.php:10]: I'm not ready to create the Till Krüss Coding Standard :)

Comment (szepeviktor) [reply stubs/runtime.php:10]: Please revert the last commit and you are ready to fly ✈️

File: stubs/runtime.php (hunk 2 above)
Comment (tillkruss) [reply stubs/runtime.php:38]: I hate PHPCS. How TF can I make it use a `phpcs.xml.dist`?

```
./vendor/bin/phpcs --config-show
Using config file:
````

Comment (szepeviktor) [reply stubs/runtime.php:38]: ~~Simply create that file in the root and the contents,~~
e.g. https://github.com/szepeviktor/phpcs-psr-12-neutron-hybrid-ruleset/blob/master/PSR12NeutronRuleset/ruleset.xml

Comment (szepeviktor) [reply stubs/runtime.php:38]: Oh no! :)
https://github.com/squizlabs/PHP_CodeSniffer/issues/2862#issuecomment-583890204

**Comment by tillkruss** [comment]
Let's work off the `ci` branch.

**Pull request closed without merging**